src/pyhf/workspace.py

```diff
@@ -743,3 +743,42 @@ def sorted(cls, workspace):
         newspec['observations'].sort(key=lambda e: e['name'])
 
         return cls(newspec)
+
+    @classmethod
+    def build(cls, model, data, name='measurement'):
+        """
+        Build a workspace from model and data.
+
+        Args:
+            model (~pyhf.pdf.Model): A model to store into a workspace
+            data (tensor): A array holding observations to store into a workspace
+            name (str): The name of the workspace measurement
+
+        Returns:
+            ~pyhf.workspace.Workspace: A new workspace object
+
+        """
+        workspace = copy.deepcopy(dict(channels=model.spec['channels']))
+        workspace['version'] = '1.0.0'
```

Comment:
@lukasheinrich This keeps on getting broken in the merge. This should use @kratsg's suggestion.
```suggestion
        workspace['version'] = utils.SCHEMA_VERSION
```